backends/aerospike.go

```diff
@@ -100,21 +100,22 @@ func (a *AerospikeBackend) Put(ctx context.Context, key string, value string, tt
 	policy := &as.WritePolicy{Expiration: uint32(ttlSeconds)}
 
 	if err := a.client.Put(policy, asKey, bins); err != nil {
-		return formatAerospikeError(err, "PUT")
+		return formatAerospikeError(err)
 	}
 
 	return nil
 }
 
-func formatAerospikeError(err error, caller ...string) error {
+func formatAerospikeError(err error) error {
 	if err != nil {
-		msg := "Aerospike"
-		for _, str := range caller {
-			if len(str) > 0 {
-				msg = fmt.Sprintf("%s %s", msg, str)
+		msg := "Aerospike "
+
+		if aerr, ok := err.(as_types.AerospikeError); ok {
+			if aerr.ResultCode() == as_types.KEY_NOT_FOUND_ERROR {
+				return KeyNotFoundError{msg}
```

Comment:
I would expect the `KeyNotFoundError` to take care of adding the trailing space on its own. Please consider the following:
```
return KeyNotFoundError{"Aerospike"}
```

```
eturn errors.New("Aerospike " + err.Error())
```